src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenScriptTests.cs

```diff
@@ -525,16 +525,17 @@ .maxstack  2
             Assert.True(methodData.Method.ReturnsVoid);
             methodData.VerifyIL(
 @"{
-  // Code size       24 (0x18)
+  // Code size       25 (0x19)
   .maxstack  1
-  .locals init (System.Runtime.CompilerServices.TaskAwaiter V_0)
+  .locals init (System.Runtime.CompilerServices.TaskAwaiter<object> V_0)
   IL_0000:  newobj     "".ctor()""
   IL_0005:  callvirt   ""System.Threading.Tasks.Task<object> <Initialize>()""
-  IL_000a:  callvirt   ""System.Runtime.CompilerServices.TaskAwaiter System.Threading.Tasks.Task.GetAwaiter()""
+  IL_000a:  callvirt   ""System.Runtime.CompilerServices.TaskAwaiter<object> System.Threading.Tasks.Task<object>.GetAwaiter()""
```

Comment:
> IL_000a:  callvirt   ""System.Runtime.CompilerServices.TaskAwaiter<object> System.Threading.Tasks.Task<object>.GetAwaiter()"" [](start = 1, length = 126)

This looks like a behavior change for a successful case. Could be a breaking change.